capstone/src/main/webapp/script.js

```diff
@@ -121,6 +121,63 @@ function drawChart() {
   document.querySelector('#chart-container').style.display = 'block';
 }
 
+/**
+ * Adds the data from the Comments servlet using async/await (the return values are used directly), and converts it to a JSON.
+ */
+async function getDataUsingAsyncAwait() {
+  // Retrieve the data from '/comments'
+  /* TODO: handle errors */
+  const response = await fetch('/comments?numComments=' + document.querySelector('#numComments').value);
+  const data = await response.json();
+  var text = "";
+  for(i = 0; i < data.length; i++){
+    text += "<b>" + data[i].name + " " + data[i].email + "</b>" + " " + "<i>" + data[i].comment + "</i><br>";
+  }
+
+  // Add the data to the page
+  const dataContainer = document.querySelector('#data-container');
+  dataContainer.style.visibility = 'visible';
+  dataContainer.style.display = 'block';
+  dataContainer.innerHTML = text;
+}
```

Comment:
Is this your part of code? I am not sure how this got added to the script.js file. I don't think this was supposed to happen. @Jamieatucker

Reply:
Yeah this is my code, it was leftover from my portfolio but I think you can remove it.